content/github/setting-up-and-managing-organizations-and-teams/verifying-your-organizations-domain.md

```diff
@@ -1,4 +1,4 @@
----
```

Comment:
TXT

Reply:
content/github/setting-up-and-managing-organizations-and-teams/verifying-your-organizations-domain.md